Formula/podman.rb

```diff
@@ -22,6 +23,11 @@ class Podman < Formula
     sha256 "e1e1bec2fc42039da1ae68d382d4560a27c04bbe2aae535837294dd6773e88e0"
   end
 
+  patch do
```

Comment:
Can we add a comment here documenting what the patch is for, and when it can be removed?